src/App.js

```diff
@@ -14,14 +14,16 @@ import MemberProfilePage from './MemberProfilePage';
 class App extends Component {
   render() {
     const { User } = this.props.data || {};
+    const CfD_logo_location =
+      process.env.PUBLIC_URL + '/images/cfd-circle-white.png';
     return (
```

Comment:
https://github.com/codefordenver/members/blob/master/docs/CreateReactApp.md line 684 seems to imply this is the best way to reference the public folder. LMK if thats not the case.

Reply:
quick note - when I use the import statement I cant get it to compile.

```./src/App.js
Module not found: You attempted to import ../public/images/cfd-circle-white.png which falls outside of the project src/ directory. Relative imports outside of src/ are not supported. You can either move it inside src/, or add a symlink to it from project's node_modules/.```

Reply:
That's interesting about the `PUBLIC_URL` param.

@KeynesYouDigIt in pull request #124 I added an `images` directory in `src` (we might want to rename it to `assets` or something), you could follow that pattern. That way we can import with the relative path.

Reply:
I think we want to use the relative import, not just because that is recommended by create react app, but because the way create-react-app is set up, using the relative import will automatically include it in the service worker so it works offline.